cmd/clusterctl/cmd/describe_cluster_test.go

```diff
@@ -294,6 +294,245 @@ func Test_TreePrefix(t *testing.T) {
 	}
 }
 
+func Test_V1Beta2TreePrefix(t *testing.T) {
+	tests := []struct {
+		name         string
+		objectTree   *tree.ObjectTree
+		expectPrefix []string
+	}{
+		{
+			name: "Conditions should get the right prefix with multiline message",
+			objectTree: func() *tree.ObjectTree {
+				root := fakeObject("root",
+					withV1Beta2Condition(metav1.Condition{
+						Type:    "Available",
+						Status:  metav1.ConditionFalse,
+						Reason:  "NotAvailable",
+						Message: "1\n2",
+					}),
+				)
+				objectTree := tree.NewObjectTree(root, tree.ObjectTreeOptions{
+					V1Beta2: true,
+				})
+
+				o1 := fakeObject("child1",
+					withV1Beta2Condition(falseV1Beta2Condition("Available", "1\n2")),
+				)
+
+				o2 := fakeObject("child2",
+					withV1Beta2Condition(falseV1Beta2Condition("Available", "1\n2")),
+				)
+				objectTree.Add(root, o1)
+				objectTree.Add(root, o2)
+				return objectTree
+			}(),
+			expectPrefix: []string{
+				"Object/root",
+				"│                                                              2",
+				"├─Object/child1",
+				"│                                                              2",
```

Comment:
I'm sure I'm missing something. Why is the 1 of `1\n2` not printed?

Reply:
Same question as this comment https://github.com/kubernetes-sigs/cluster-api/pull/11781#discussion_r1942604097 :)

Reply:
The real output looks like

```
Object/root              Available: False  NotAvailable  292y  1  
│                                                              2  
├─Object/child1          Available: False  NotAvailable  292y  1  
│                                                              2  
└─Object/child2          Available: False  NotAvailable  292y  1  
                                                               2  
```

Reply:
Pushed it, I also had to fix some test titles as they still had some random characters during test creation

Reply:
Also aligned all tests to have the status and reason. IMO this makes it more clear how it should look like

Reply:
Thank you!